Tests/FTPropertyWrappersTests/KeychainCoderTest.swift

```diff
@@ -0,0 +1,151 @@
+import XCTest
+@testable import FTPropertyWrappers
+
+final class KeychainCoderTest: XCTestCase {
+
+    private class WrapperClass<T: Codable & Equatable>{
+
+        let wrapped: T
+        init(wrapped: T) {
+            self.wrapped = wrapped
+        }
+        func testCoding() throws {
+            let coder = KeychainCoding()
+            let data = try coder.encode(wrapped)
+            XCTAssertEqual(try coder.decode(T.self, from: data), wrapped)
+        }
+    }
```

Comment:
There is no need for this wrapper class in tests. I think the less abstraction in test the better. I would use only the helper function.

Reply:
Having this type is a result of development proces, during which I found out, that simply having overloaded function is not a solution when called from within a generic type instance. Getting rid of this wrapper may result in coding working in tests but failing in Keychain (as it did during development). Calling overloaded method from a non-generic scope will work. Calling overloaded method from a generic scope won't work as expected.